addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -135,7 +135,7 @@ patch(PosStore.prototype, {
             isDraftOrder = true;
         }
 
-        if (this.get_order()?.finalized) {
+        if (this.get_order()?.finalized && this.mainScreen.component !== ReceiptScreen) {
```

Comment:
You should check the TipScreen also no ?

Reply:
oh thats true I forgot this one